tests/test_smoke.py

```diff
@@ -2351,6 +2351,18 @@ def tmp_scratch_storage_obj(self, tmp_bucket_name):
         # Stores must be added in the test.
         yield from self.yield_storage_object(name=tmp_bucket_name)
 
+    @pytest.fixture
+    def tmp_multiple_scratch_storage_obj(self):
+        # Creates a list of 5 storage objects with no source to create
+        # multiple scratch storages.
+        # Stores for each object in the list must be added in the test.
+        storage_mult_obj = []
+        for _ in range(5):
+            timestamp = str(time.time()).replace('.', '')
+            store_obj = storage_lib.Storage(name=f'sky-test-{timestamp}')
+            storage_mult_obj.append(store_obj)
+        yield storage_mult_obj
```

Comment:
Is there some way to get storage objects from `yield_storage_object` here? That was deletion after test is handled by that code. If not, we should add deletion code after the yield statement here (refer to `yield_storage_object`).

Reply:
We are unable to use multiple yield statements in fixture as they are not supported. Hence, had to resort to this specific way of implementation.  I am adding the deletion code after the yield statement as suggested.